packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -95,8 +95,8 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     marshaller,
   );
 
-  const makeStatusNode = () => E(storageNode).makeChildNode(STATUS_NODE);
-  const statusManager = prepareStatusManager(zone, makeStatusNode);
+  const statusNode = E(storageNode).makeChildNode(STATUS_NODE);
```

Comment:
I'm confused about line 178: "Define all kinds above this line. Keep remote calls below."

Seems like this would go against that because it's a remote call. Does it matter? The `makeStatusNode` was already kind of a gray area in that it wasn't called immediately.

Reply:
That comment really meant "keep remote call _awaits_ below". We'll need to improve the comment and patterns for contract developers about that, but it's out of scope for this PR.

Reply:
Got it, thanks for clarifying.